webapp/channels/src/components/admin_console/license_settings/starter_edition/starter_right_panel.tsx

```diff
@@ -9,10 +9,12 @@ import SetupSystemSvg from 'components/common/svg_images_components/setup_system
 
 const StarterRightPanel = () => {
     const upgradeAdvantages = [
-        'OneLogin/ADFS SAML 2.0',
-        'OpenID Connect',
-        'Office365 suite integration',
-        'Read-only announcement channels',
+        'Attribute-based access control',
```

Comment:
Same question here about translation. Ignore if it's a non-issue

Reply:
Also updated to be translatable